core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/impl/LoadSequencerWorkflow.java

```diff
@@ -0,0 +1,60 @@
+package uk.ac.bbsrc.tgac.miso.core.data.workflow.impl;
+
+import static uk.ac.bbsrc.tgac.miso.core.data.workflow.Workflow.WorkflowName.LOAD_SEQUENCER;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.AbstractWorkflow;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.ProgressStep;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.WorkflowExecutor;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.WorkflowStep;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.WorkflowStepPrompt;
+
+public class LoadSequencerWorkflow extends AbstractWorkflow {
+  @Override
+  protected List<WorkflowStep> getCompletedSteps() {
+    return Collections.emptyList();
+  }
+
+  @Override
+  protected WorkflowName getWorkflowName() {
+    return LOAD_SEQUENCER;
+  }
+
+  @Override
+  public WorkflowStepPrompt getNextStep() {
+    return null;
+  }
+
+  @Override
+  public WorkflowStepPrompt getStep(int stepNumber) {
+    return null;
+  }
+
+  @Override
+  public boolean isComplete() {
+    return false;
+  }
+
+  @Override
+  public void processInput(ProgressStep step) {
+    throw new UnsupportedOperationException("Not implemented yet");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Not implemented yet" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)